matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/Session.kt

```diff
@@ -47,6 +49,18 @@ interface Session :
     @MainThread
     fun open()
 
+    /**
+     * This method start the sync thread.
+     */
+    @MainThread
+    fun start()
```

Comment:
Why not calling them startSync and stopSync ? It'll allow to handle notifications?

Reply:
Done